homeassistant/components/onboarding/__init__.py

```diff
@@ -0,0 +1,42 @@
+"""Component to help onboard new users."""
+from homeassistant.core import callback
+from homeassistant.loader import bind_hass
+
+from .const import STEPS, DOMAIN
+
+DEPENDENCIES = ['http']
+STORAGE_KEY = DOMAIN
+STORAGE_VERSION = 1
+
+
+@bind_hass
+@callback
+def async_is_onboarded(hass):
+    """Return if Home Assistant has been onboarded."""
+    # Temporarily: if auth not active, always set onboarded=True
+    if not hass.auth.active:
+        return True
+
+    return hass.data.get(DOMAIN, True)
```

Comment:
Should check if owner already created
```
if hass.auth.is_onboarded():
    return True
```

Reply:
We do those checks inside the `async_setup`. We're not checking explicitly for owner, but instead check if the user step is done. There can be many steps in the future.